src/auth/auth.cpp

```diff
@@ -24,46 +25,84 @@
 #include "utils/settings.hpp"
 #include "utils/string.hpp"
 
-DEFINE_VALIDATED_string(auth_module_executable, "", "Absolute path to the auth module executable that should be used.",
+namespace memgraph {
+std::unordered_map<std::string, std::string> ModuleMappingsToMap(const std::string &module_mappings) {
+  std::unordered_map<std::string, std::string> module_per_scheme;
+  for (const auto &mapping : utils::Split(module_mappings, ";")) {
+    const auto module_and_scheme = utils::Split(mapping, ":");
+    if (module_and_scheme.size() != 2) {
+      throw auth::AuthException("Entries in the auth module mapping follow the \"auth_schema: module_path\" syntax!");
+    }
+    const auto scheme_name = std::string{utils::Trim(module_and_scheme[0])};
+    const auto module_path = std::string{utils::Trim(module_and_scheme[1])};
+    module_per_scheme.emplace(scheme_name, module_path);
```

Comment:
That’s right

Reply:
@tonilastre does this cover all the use-cases you're interested in?
I know you were talking about having multiple LDAP modules. Could all those also be defined in this way?

Reply:
Just to confirm the above re scheme, the scheme contains both the protocol and identity provider, so it is a single module per scheme.

Re: LDAP, not sure because I haven't worked with that + Memgraph.

Reply:
Resolving since 1) Toni has confirmed the scheme information 2) multiple LDAP modules are out of scope here